Samsung/LPVS
# Add History page.

## Description

LPVS Front-End History Page.
History Page provides a history of generated PRs.
- My Pull Request - History of PR I created
- My Repo PR - History of PR created in My Repository
- My Org PR - History of PR created in my organization

Result, Dashboard pages will be uploaded.

## Type of change


# How Has This Been Tested?
1. Update environment variables of `docker-compose.yml` file.
2. Build the lpvs container : `docker compose build`
3. Run containers: `docker compose up -d`
4. See the lpvs home page with any browser: `http://localhost:7896`

For using social login with Naver, Kakao, and Google, fill the necessary information:
- Update the `application.properties` located in `src/main/resources`:
Enter client-id and client-secret. (Kakao doesn't require a client-secret.)
- Update the `.env` file located in `frontend`:
Enter your REST_API_KEY.

The values required to implement the login function can be obtained from the link below.
Naver: https://developers.naver.com/products/login/api/api.md
Kakao: https://developers.kakao.com/docs/latest/ko/kakaologin/common
Google: https://cloud.google.com/identity-platform/docs/web/google

![history](https://github.com/Samsung/LPVS/assets/57388014/e6fef320-eeae-45d9-a983-4ed46f077048)
![history_repo](https://github.com/Samsung/LPVS/assets/57388014/577e4c7b-83d0-4887-a7c9-f75cccce0fc1)



**Test Configuration**:
* Java: v11
* LPVS Release: v1.1.0

# Checklist:
